StripeFinancialConnections/StripeFinancialConnections/Source/Analytics/FinancialConnectionsAnalyticsClient.swift

```diff
@@ -111,7 +111,16 @@ extension FinancialConnectionsAnalyticsClient {
             parameters["code"] = apiError.code
         } else {
             parameters["error_type"] = (error as NSError).domain
-            parameters["error_message"] = (error as NSError).localizedDescription
+            parameters["error_message"] = {
+                if let sheetError = error as? FinancialConnectionsSheetError {
```

Comment:
seems like the "design" of `FinancialConnectionsSheetError` is that it will print an "ambiguous" `localizedDescription`, so here I am maintaining those principles, and explicitly unwrapping the error to get the debug description

this helps to fix an issue where debugging `FinancialConnectionsSheetError` with Splunk was actually difficult (all sheet errors start looking the same)

Reply:
thank you! This makes sense.